services/s3/src/it/java/software/amazon/awssdk/services/s3/OperationsWithNonStandardResponsesIntegrationTest.java

```diff
@@ -61,6 +65,16 @@ public void getBucketLocationReturnsAResult() {
         assertThat(s3.getBucketLocation(request).locationConstraint()).isEqualTo(BucketLocationConstraint.US_WEST_2);
     }
 
+    @Test
+    public void getBucketLocation_withArn_throwsDetailedErrorMessage2() {
+        String bucketArn = "arn:aws:s3:::mybucket";
+        SdkClientException exception = assertThrows(SdkClientException.class,
+                                                    () -> {s3.getBucketLocation(b-> b.bucket(bucketArn).build());});
```

Comment:
Can you clean up the formatting of this line, by either removing the brackets/semicolon or utilize more lines?

Reply:
removed brackets/semicolon